code/game/objects/items/weapons/tape.dm

```diff
@@ -4,6 +4,13 @@
 	icon = 'icons/obj/bureaucracy.dmi'
 	icon_state = "taperoll"
 	w_class = 1
+	var/charge = 10
+
+/obj/item/weapon/tape_roll/proc/take_charge(var/mob/user,var/amount)
```

Comment:
This is a pretty ridiculous nerf to tape. I'm almost inclined to veto the PR entirely over this.

Reply:
There was a hiest round earlier today where the entire crew were captured via a single roll of tape. ERT ended up getting headgibbed by tapethrow spam. I genuinely think that this tape strat needs to be addressed via a balance change such as this.